src/lib/commandclass/ColorSwitchCC.ts

```diff
@@ -131,13 +137,6 @@ function getTargetColorValueID(
 	};
 }
 
-export interface ColorSwitchGetResult {
```

Comment:
This was intended to be the return type of ColorSwitchCCAPI.get, looks like I forgot to specify it in my earlier MR

Reply:
I've resorted to turning off the explicit return type check for the more complicated API methods.

Reply:
Its still useful to have, so that typescript users can know what to expect from the return type.  At least keep the interface and let typescript users cast it on their own.

Reply:
The information is still there in the generated .d.ts files - its just not explicitly specified in the source file.
![grafik](https://user-images.githubusercontent.com/17641229/79369275-fbfd1580-7f50-11ea-9d7b-e3502570e5f4.png)